### PR TITLE
Replace deprecated addCase calls with case

### PR DESCRIPTION
**Pull request by jadelbe418**

### CODE REVIEW

**Review by Arhell — APPROVED** [review]

File: en/orm/query-builder.rst [hunk 1]
```diff
@@ -621,7 +621,7 @@ size, we could do the following::
 
     $query = $cities->find()
         ->where(function (QueryExpression $exp, Query $q) {
-            return $exp->addCase(
+            return $exp->case(
```
Comment (othercorey) [inline en/orm/query-builder.rst:624]: This example needs to be updated to work with case().

File: en/orm/query-builder.rst [hunk 2]
```diff
@@ -642,7 +642,7 @@ automatically produce an ``if .. then .. else`` statement::
...
```
Comment (othercorey) [inline en/orm/query-builder.rst:645]: This example needs to be updated too and ``addCase`` changed above.

File: en/orm/query-builder.rst (hunk 1 above)
Comment (othercorey) [reply en/orm/query-builder.rst:624]: @ndm2 Can you take a look at this?

**Comment by ndm2** [comment]
This should not be changed, these examples are for CakePHP pre 4.3, where `case()` doesn't exist.

Maybe the sentence "_Prior to 4.3.0, you would need to use:_" should be made into a note, or a note should be there additionally, so that it's easier to spot.

**Comment by othercorey** [comment]
@jadelbe418 Do you want to replace this with a comment as suggested by @ndm2 ?

**Comment by jadelbe418** [comment]
@othercorey Sure, I can do that. I can look at it again and make the changes by the end of the day.

**Pull request closed without merging**